src/openklant/components/klantinteracties/tests/test_validators.py

```diff
@@ -330,10 +357,118 @@ def test_not_allowed_cases(self):
         ]
         for case in invalid_cases:
             with self.assertRaises(ValidationError):
-                validator = PartijIdentificatorValidator(
-                    code_register=case[0],
-                    code_objecttype=case[1],
-                    code_soort_object_id=case[2],
-                    object_id=case[3],
+                validator = PartijIdentificatorTypesValidator(
+                    partij_identificator={
+                        "code_register": case[0],
+                        "code_objecttype": case[1],
+                        "code_soort_object_id": case[2],
+                        "object_id": case[3],
+                    }
                 )
                 validator.validate()
+
+
+class PartijIdentificatorUniquenessValidatorTests(TestCase):
+
+    def test_valid_global_uniqueness(self):
+        PartijIdentificatorUniquenessValidator(
+            partij_identificator={
+                "code_objecttype": PartijIdentificatorCodeObjectType.natuurlijk_persoon.value,
+                "code_soort_object_id": PartijIdentificatorCodeSoortObjectId.bsn.value,
+                "object_id": "296648875",
+                "code_register": PartijIdentificatorCodeRegister.brp.value,
+            },
+            sub_identificator_van=None,
+            instance=None,
+        ).validate()
```

Comment:
can you also add an assertion here where you expect it to throw an error based on uniqueness?